### PR TITLE
fix broken image in pdf report

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: 
* [XSUP-16759](https://jira-hq.paloaltonetworks.local/browse/XSUP-16759)
* [CIAC-4399](https://jira-hq.paloaltonetworks.local/browse/CIAC-4399)

## Description
* fixing broken image
* fixing logging to stderr
* fixing is_demisto_version_ge

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by glicht** [comment]
TDD: failed unit test after fixing return status when there is an error opening the file: https://app.circleci.com/pipelines/github/demisto/content/200620/workflows/b3e2f8df-b403-4a38-9ee4-d5b13ee243b3/jobs/586541

**Comment by glicht** [comment]
TDD for CSP: https://app.circleci.com/pipelines/github/demisto/content/200806/workflows/e6a9330e-4d81-4704-92c4-e39a4bb1a9a6/jobs/586829

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -7884,12 +7883,29 @@ def is_demisto_version_ge(version, build_number=''):
...
     """
+
+    def versionzfill(v):
+        """ Split version on . and zfill. So we can compare 6.10 to 6.5 properly.
+
+        :type version: ``str``
+        :param version: Version to check
+
+        :return: True if running within a Server version greater or equal than the passed version
+        :rtype: ``bool``
```
Comment (ilappe) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:7894]: looks like doc comment copied form another place 
versionzfill is return tuple

**Review by ilappe — APPROVED** [review]

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py (hunk 1 above)
Comment (glicht) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:7894]: Correct. Will fix

**Pull request merged**